### ISSUE TITLE
[Bug] "Internal Error" (version 0.8.0b1) when adding buttons to radio box [sf#232]

### ISSUE THREAD

**Issue by DietmarSchwertberger** [DietmarSchwertberger/wxGlade#303]
**Reported by rsonnenfeld on 2017-12-30 20:35 UTC**
Greetings Maintainers!  To get more comfortable with wxglade, I have slavishly followed the directions here: http://wxglade.sourceforge.net/docs/tutorial.html to create a calculator app.
Everything was perfect up till the step where I was asked to add radio buttons for +-*/.

As you can see from screenshot, the buttons DID get added (and the designer window shows
them correctly) but every press of the "Add" (or "Remove") key resulted in the following error.

The summary said "type.exceptions "Type Error" and "This is a bug, report it" even though
the detailed description says ERROR: None.  The code appears to be working, apart from this
obnoxious message.  I will update the report when I finish the calculator project and let you know
if the radio buttons actually created functional code.

>  ERROR   : None
An unexpected error occurred!

Exception type:      <type 'exceptions.TypeError'>
Exception details:   Required argument 'col' (pos 3) not found

Application stack traceback:
  File "/home/richard/aainfors/Tech/python/wxglade0.8/new_properties.py", line 2461, in add_row
    self.grid.GoToCell( (len(values)-1,self.cur_col) )
  File "/usr/lib/python2.7/dist-packages/wx-3.0-gtk2/wx/grid.py", line 1428, in GoToCell
    return _grid.Grid_GoToCell(*args, **kwargs)
    
    Thanks for the excellent tutorial BTW, and version 0.8 seems to be quite a bit easier to work with than 0.6 was.

**Comment by DietmarSchwertberger** [comment]
**Commented by rsonnenfeld on 2017-12-30 20:36 UTC**
I am working on an update to the tutorial BTW just to make it a little clearer.
Can I e-mail that to someone?

**Comment by DietmarSchwertberger** [comment]
**Updated by dschwertberger on 2017-12-30 20:46 UTC**
- **status**: open --> accepted
- **assigned_to**: Dietmar Schwertberger

**Comment by DietmarSchwertberger** [comment]
**Updated by dschwertberger on 2017-12-30 22:37 UTC**
- **status**: accepted --> pending

**Comment by DietmarSchwertberger** [comment]
**Commented by dschwertberger on 2017-12-30 22:37 UTC**
Thanks for reporting.
I did some changes recently on grid/list properties to make editing smoother.
I'm mainly testing on wxPython Phoenix, so I did not notice that Grid.GoToCell is different with Classic and not available on 2.8.

The fix is now in the repository and will be in revision 0.8.0b2.
Nowadays I recommend to switch to Phoenix. It has been quite usable for more than one year now.

See separate PM for the tutorial.
